pkg/scalers/rabbitmq_scaler.go

```diff
@@ -715,3 +721,11 @@ func (s *rabbitMQScaler) anonymizeRabbitMQError(err error) error {
 	errorMessage := fmt.Sprintf("error inspecting rabbitMQ: %s", err)
 	return fmt.Errorf(rabbitMQAnonymizePattern.ReplaceAllString(errorMessage, "user:password@"))
 }
+
+// connectionName is used to provide a deterministic AMQP connection name when
+// connecting to RabbitMQ. As far as I can tell, there are no restrictions
+// on the connection name:
+// https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/amqp_client/src/amqp_connection.erl#L185-L196
```

Comment:
It uses `longstr` type which has no limit that affects us
```suggestion
// connectionName is used to provide a deterministic AMQP connection name when
// connecting to RabbitMQ
```